### DISCUSSION TITLE
Welcome to codemeta Discussions!

### DISCUSSION THREAD

**Discussion by moranegg** [codemeta/codemeta#304] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by mbjones** [comment]
Thanks for organizing, @moranegg!

Hi, I'm Matt Jones, working at [NCEAS](https://nceas.ucsb.edu) at UC Santa Barbara on open data science infrastructure, including the [DataONE](https://dataone.org) network and repositories like the [Arctic Data Center](https://arcticdata.io) and the [KNB Data Repository](https://knb.ecoinformatics.org).  I've been involved in many metadata standards over the years, and have contributed extensively to [EML](https://eml.ecoinformatics.org), FGDC BDP, the https://science-on-schema.org effort for Dataset metadata guidelines, and CodeMeta for software. For CodeMeta, I was involved in the initial creation of the CodeMeta terms vocabulary, helped organize workshops that led to our implementation of CodeMeta 1.0, and then helped map CodeMeta terms to schema.org, resulting in CodeMeta 2.0. I'm looking forward to the continued growth of this community and uptake of CodeMeta metadata conventions.

**Comment by moranegg** [comment]
Thanks @mbjones for starting the ball rolling!

Hi, I'm Morane Gruenpeter, working at [Software Heritage](https://www.softwareheritage.org/) and based in Paris, France in the [Inria](https://www.inria.fr/en) research center. I have joined Software Heritage in 2017 to tackle the challenge of describing software source code with metadata as a software engineer. Early on I have discovered the CodeMeta initiative and based our solution on the CodeMeta crosswalk table. Today I have a coordination and project management role to setup partnerships with scholarly infrastructures to develop tools and services for archival, reference, description, and citation of research software artifacts. A few of these partnerships are part of the European project [FAIRCORE4EOSC](https://faircore4eosc.eu/eosc-core-components/eosc-research-software-apis-and-connectors-rsac).

**Comment by dgarijo** [comment]
Hello,
I am Daniel Garijo, a distinguished researcher at Universidad Politécnica de Madrid ([Ontology Engineering Group](https://oeg.fi.upm.es/)). I have worked describing research software for a while, first in the [OntoSoft](https://ontosoft.org/ontology/software/) vocabulary (part of the crosswalk table) and afterwards with the [Software Description Ontology](https://w3id.org/okn/o/sd/) which extends schema.org. My interest now is in mining research software metadata automatically, aligning it with Codemeta, to reduce the effort needed in maintaining software records. I would like to participate in Codemeta to extend it with more metadata :)

**Comment by bandrow** [comment]
Hello Anita Bandrowski here from the SciCrunch Registry https://scicrunch.org/resources/data/source/nlx_144509-1/search

**Comment by owlice** [comment]
I'm Alice Allen; I'm the editor of the Astrophysics Source Code Library (ascl.net).

**Comment by ljgarcia** [comment]
Hello, I am Jael, ZB MED, interested in metadata for research artifacts.

**Comment by Aidajafarbigloo** [comment]
Hello, I'm Aida, from OFFIS - Institut für Informatik and Oldenburg University. I'm interested in and working on research software metadata mappings.

**Comment by JSadowska** [comment]
Hello, I'm Jozefina Sadowska from Inria France. I'm a data and open-science librarian, and I'm interested in research-software description, preservation, and metadata mappings.

**Comment by jmaassen** [comment]
Hi, I'm Jason Maassen from the Netherlands eScience Center and project lead of the [Research Software Directory](https://research-software-directory.org). I'm interested in using CodeMeta as an exchange format that allows us to import or export information about research software, in particular information about how software links to other research outputs and activities.

**Comment by lindangulopez** [comment]
Hi I'm [Linda Angulo Lopez](https://www.linkedin.com/in/lindangulopez/) from Software Heritage.
